src/java/org/apache/cassandra/db/Keyspace.java

```diff
@@ -527,6 +527,7 @@ public OpOrder.Barrier stopMutations()
         assert writeBarrier == null : "Keyspace has already been closed to mutations";
         writeBarrier = writeOrder.newBarrier();
         writeBarrier.issue();
+        logger.debug("BARRIER: Issuing new barrier for {}", getName(), new Throwable());
```

Comment:
This doesn't seem related to the changes.

Reply:
yes, just ignore it pls